fireant/tests/slicer/test_managers.py

```diff
@@ -163,6 +164,48 @@ def test_transform_highcharts_require_cont_dim_on_slicer_with_no_dims(self, mock
         with self.assertRaises(TransformationException):
             self._test_transform(self.slicer.highcharts.line_chart, mock_transform, request)
 
+    @patch.object(HighchartsAreaTransformer, 'transform')
+    def test_transform_highcharts_area_chart(self, mock_transform):
+        request = {
+            'metrics': ['foo', 'bar'],
+            'dimensions': ['cont'],
+            'metric_filters': (), 'dimension_filters': (),
```

Comment:
Interesting, I did not know we could declare tuples like that in dicts. I always assumed a comma was necessary.